central/image/datastore/store/postgres/store.go

```diff
@@ -1107,6 +1118,9 @@ func (s *storeImpl) retryableGetMany(ctx context.Context, ids []string) ([]*stor
 	resultsByID := make(map[string]*storage.Image)
 	for _, id := range ids {
 		msg, found, err := s.getFullImage(ctx, tx, id)
+		if err := tx.Commit(ctx); err != nil {
```

Comment:
this goes with @connorgorman comment on the node file.  I think we want this outside the loop for consistency and do all the queries under the same transaction.